src/basic/audit-util.c

```diff
@@ -93,7 +93,7 @@ bool use_audit(void) {
 
                 fd = socket(AF_NETLINK, SOCK_RAW|SOCK_CLOEXEC|SOCK_NONBLOCK, NETLINK_AUDIT);
                 if (fd < 0)
-                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT;
+                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT && errno != EPERM;
```

Comment:
I figure this patch is OK, but I'd like one little change: I think it's OK to silently turn off audit if the kernel doesn't support audit at all. But in the case of a permission problem we should log about this, at least at debug level. Hence, please add a quick log_debug_errno() invocation here (only for the EPERM case), explaining that audit is turned off now, and why. This way, people won't be annoyed by this by default, and the right thing happens if for some reason we lack the audit privs. However people who want to debug this will still grok what's going on as soonas debug logging is enabled.